mesonbuild/interpreter.py

```diff
@@ -2232,6 +2233,14 @@ def add_devenv_method(self, args: T.Union[str, list, dict, EnvironmentVariablesH
             env = EnvironmentVariablesHolder(env)
         self.build.devenv.append(env.held_object)
 
+    @noPosargs
+    @permittedKwargs({'unbuffered'})
+    def get_command(self, args, kwargs):
+        unbuffered = kwargs.get('check', False)
```

Comment:
```suggestion
        unbuffered = kwargs.get('unbuffered', False)
```

I'm not sure we need to expose that, what does it do?

Reply:
It adds the `-u` flag to the call. I.e. `[ '/usr/bin/python3', '/foo/meson.py' ]` becomes `['/usr/bin/python3', '-u', '/foo/meson.py']`. Not sure if that is necessary. I included it, since the original method provides it. I am happy to remove that part.